### PR TITLE
Integrates n-messages

### PR DESCRIPTION
**Pull request by ifyio**
cc/ @i-like-robots @wheresrhys

### CODE REVIEW

**Comment by ifyio** [comment]
removing the .idea folder

File: configure.scss [hunk 1]
```diff
@@ -11,6 +11,7 @@ $_n-ui-features: (
 	grid: true,
 	header: true,
 	icons: null,
+	messages: true,
```
Comment (i-like-robots) [inline configure.scss]: should be false I reckon

**Comment by i-like-robots** [comment]
Looks fine to me - is the plan to copy over the messages code into this repo?

File: configure.scss (hunk 1 above)
Comment (wheresrhys) [inline configure.scss]: should be `false` here, otherwise they would get initialised on error pages, opt out form etc

**Comment by wheresrhys** [comment]
Looks good (aside from 1 comment). 

Will this in general only be used for promotional messages? If it will, could it be renamed to promo-messages or something along those lines, to make clearer the distinction between it and notifications.

**Comment by ifyio** [comment]
@i-like-robots Yes it is. Eventually.

**Comment by ifyio** [comment]
@wheresrhys yes that's a good idea. Will do that now

**Pull request merged**